Garmin.Connect/GarminConnectClient.cs

```diff
@@ -29,6 +31,8 @@ public partial class GarminConnectClient : IGarminConnectClient
     private const string DeviceListUrl = "/device-service/deviceregistration/devices";
     private const string DeviceServiceUrl = "/device-service/deviceservice/";
     private const string GearUrl = "/gear-service/gear/";
+    private const string WorkoutUrl = "/workout-service/workout/";
+    private const string WorkoutsUrl = "/workout-service/workouts";
```

Comment:
Ok, I am starting investigate them now

Reply:
I added method `ScheduleWorkout` but did not `SendToDevice` because it is more complicated and better add it with new PR with devices re-test #60 
Anyway after `ScheduleWorkout` you can press sync manually in app and get new calendar on your device.

And requested few new calendar methods because that need for removing workouts #58